integration-tests/templates/_Cargo.toml

```diff
@@ -21,3 +23,6 @@ opt-level = "z"
 lto = true
 debug = false
 panic = "abort"
+
+[patch.crates-io]
+borsh = { git = 'https://github.com/near/borsh-rs.git', branch = "slice_schema_container" }
```

Comment:
TODO: remove after [borsh#245](https://github.com/near/borsh-rs/pull/245) is merged